### PR TITLE
Implements logic to disable page scroll when search is open on mobile

### PR DESCRIPTION
**Pull request by gkozlowaski**

### CODE REVIEW

File: src/components/DefaultTemplate/DefaultTemplate.js [hunk 1]
```diff
@@ -10,7 +10,11 @@ const Placeholder = stores.ComponentStore.state.getIn(['Placeholder@vtex.storefr
 
 class DefaultTemplate extends React.Component {
   componentWillMount() {
+<<<<<<< HEAD
```
Comment (amoreira) [inline src/components/DefaultTemplate/DefaultTemplate.js]: Ouch! It seems that the rebase got you!

File: src/components/DefaultTemplate/DefaultTemplate.js [hunk 2]
```diff
@@ -41,12 +63,30 @@ class DefaultTemplate extends React.Component {
...
+          transitionEnterTimeout={200}
+          transitionLeaveTimeout={250}
+        >
+          { loading }
+        </ReactCSSTransitionGroup>
+        <Header
+          pageScrollOn={this.pageScrollOn}
+          pageScrollOff={this.pageScrollOff}
+        />
+>>>>>>> Implements functions to disable scroll on page
```
Comment (amoreira) [inline src/components/DefaultTemplate/DefaultTemplate.js]: Same here. =(

File: src/components/DefaultTemplate/DefaultTemplate.js [hunk 3]
```diff
@@ -10,12 +10,8 @@ const Placeholder = stores.ComponentStore.state.getIn(['Placeholder@vtex.storefr
 
 class DefaultTemplate extends React.Component {
   componentWillMount() {
-<<<<<<< HEAD
```
Comment (amoreira) [inline src/components/DefaultTemplate/DefaultTemplate.js]: You could have used the 'git rebase -i < commit> command (https://git-scm.com/docs/git-rebase).

**Comment by tamorim** [comment]
@gkozlowaski Is really necessary to do all this work and pass this props just do disable the scroll?
Couldn't you target the `body` on the `search-bar` to do this?
Like:
```javascript
componentDidMount() {
  document.body.style.position = 'fixed';
  document.body.style.overflow = 'hidden';
}

componentWillUnmount() {
  document.body.style.position = 'static';
  document.body.style.overflow = 'auto';
}
```

Ou se quiser pode até guardar o estado anterior do `body` pra voltar de acordo.

**Comment by gkozlowaski** [comment]
Following @tamorim advise, there's a better approach that doesn't require changes inside theme code. Closing this pull request because of that.

**Pull request closed without merging**